specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -1334,6 +1334,64 @@
         "apiKey"
       ]
     },
+    "AzureMLServiceLinkedService": {
```

Comment:
@LijuanZ I was trying to find an API endpoint using this model, it seems model is not referenced any where. Is that intentional?

Reply:
@anuchandy This is one new linked service resource type, just similar to all other linked services defined in LinkedService.json file. API endpoint "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/linkedservices/{linkedServiceName}" references general linkedService type